src/pages/modules/components/ModuleAccordion.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import {
+  Button,
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+} from "@mui/material";
+import { ExpandMore, Add } from "@mui/icons-material";
+import { useAppSelector } from "../../../app/common/hooks";
+import { ModuleMenu, ProblemButtons } from "./";
+import { styled } from "@mui/material/styles";
+import { grey } from "@mui/material/colors";
+
+const Module = styled(Accordion)({
+  width: "100%",
+});
+
+const ModuleSummary = styled(AccordionSummary)({
+  backgroundColor: "rgba(0, 0, 0, .1)", // background of module
+  borderBottom: "1px solid rgba(0, 0, 0, .125)", // bottom line of module
+});
+
+const ModuleContent = styled(AccordionDetails)(({ theme }) => ({
+  display: "flex",
+  padding: theme.spacing(1),
+  paddingLeft: theme.spacing(2),
+  backgroundColor: "rgba(0, 0, 0, .02)",
+  borderBottom: "1px solid rgba(0, 0, 0, .125)",
+}));
+
+const ProblemTitle = styled(Typography)({
+  marginTop: "auto",
+  marginBottom: "auto",
+});
+
+const NewProblemButton = styled(Button)(({ theme }) => ({
+  marginLeft: "auto",
+  marginRight: theme.spacing(1),
+  color: grey["A700"],
```

Comment:
nit: why this specific grey? if you use it frequently, we should probably put it in some constant

Reply:
The grey is to make it look like the wireframe, I needed to add the grey color by hand since the default color of the mui components is blue. 

I do not plan to make this permanent, so adding constants to access this will be when we determine what the theme will be like. So I will be waiting for the theme provider.